### ISSUE TITLE
Improve dependency management output

### ISSUE THREAD

**Issue by cemsbr** [kytos/kytos-utils#135]
Notice that there are also changes in the spaces that should be applied to local NApp management as well.

For the outputs below, consider `$ kytos napps install kytos/of_lldp` as an example.

All NApps already installed
----------------------------------

Actual
```
INFO  NApp kytos/of_lldp:
WARNING   Napp already Installed.
INFO    Enabled.
INFO  NApp kytos/of_core:
INFO    Enabled.
INFO  Installing Dependencies:
INFO  NApp kytos/of_core:
WARNING   Napp already Installed.
INFO    Enabled.
```

Expected
```
INFO     NApp kytos/of_lldp:
WARNING    Napp already enabled.
```

No NApp installed
-----------------------
Actual
```
INFO  NApp kytos/of_lldp:
INFO    Searching local NApp...
INFO    Not found. Downloading from NApps Server...
INFO    Downloaded and installed.
INFO    Enabling...
INFO    Enabled.
INFO  NApp kytos/of_core:
INFO    Enabling...
ERROR   Install NApp kytos/of_core first.
INFO    Installed.
INFO  Installing Dependencies:
INFO  NApp kytos/of_core:
INFO    Searching local NApp...
INFO    Not found. Downloading from NApps Server...
INFO    Downloaded and installed.
INFO    Enabling...
INFO    Enabled.
INFO    Installed.
```
Expected
```
INFO     NApp kytos/of_lldp:
INFO       Searching local NApp...
INFO       Not found. Downloading from NApps Server...
INFO       Downloaded and installed.
INFO       Enabling...
INFO       Enabled.
INFO   Installing Dependencies:
INFO     NApp kytos/of_core:
INFO       Searching local NApp...
INFO       Not found. Downloading from NApps Server...
INFO       Downloaded and installed.
INFO       Enabling...
INFO       Enabled.
```

Dependency already installed
-------------------------------------
Actual
```
INFO  NApp kytos/of_lldp:
INFO    Searching local NApp...
INFO    Not found. Downloading from NApps Server...
INFO    Downloaded and installed.
INFO    Enabling...
INFO    Enabled.
INFO  NApp kytos/of_core:
INFO    Enabled.
INFO    Installed.
INFO  Installing Dependencies:
INFO  NApp kytos/of_core:
WARNING   Napp already Installed.
INFO    Enabled.
```

Expected
```
INFO     NApp kytos/of_lldp:
INFO       Searching local NApp...
INFO       Not found. Downloading from NApps Server...
INFO       Downloaded and installed.
INFO       Enabling...
INFO       Enabled.
INFO   Installing Dependencies:
INFO     NApp kytos/of_core:
WARNING    Napp already enabled.
```

**Comment by cemsbr** [comment]
@macartur, all the cases have different outputs from the ones specified in the issue description. Compare them carefully, please.

**Comment by macartur** [comment]
We was closed by #145
